### ISSUE TITLE
Fails to build with ngcc in vscode with angular language service

### ISSUE THREAD

**Issue by lincolnthree** [rx-angular/rx-angular#552]
# Description

```
[ng] Build at: 2021-02-23T05:27:07.304Z - Hash: 4d037ff176977e7829cf - Time: 2850ms
[ng] Error: ./node_modules/@rx-angular/template/__ivy_ngcc__/fesm2015/rx-angular-template.js
[ng] Module build failed (from ./node_modules/@ngtools/webpack/src/ivy/index.js):
[ng] Error: ENOENT: no such file or directory, open '/Users/lincoln/projects/maritlabs/topdecked-unified/node_modules/@rx-angular/template/__ivy_ngcc__/fesm2015/rx-angular-template.js'
```

# Steps to Reproduce the Issue

1. Download example
2. cd example
3. npm i
4. open in vscode with language service

[example.zip](https://github.com/rx-angular/rx-angular/files/6029475/example.zip)

# Environment

# Related to Other Issues

Related to: https://github.com/angular/vscode-ng-language-service/issues/1144

# Tasks to Resolve This

Edit package.json to point to a `main:` source file that exists within the rx-angular distribution.

# Notes

They claim the issues is that package.json points to a file that does not exist:

https://github.com/angular/vscode-ng-language-service/issues/1144

**Comment by lincolnthree** [comment]
https://github.com/angular/vscode-ng-language-service/issues/1144#issuecomment-784414486

**Comment by hoebbelsB** [comment]
ty!

**Comment by hoebbelsB** [comment]
i've just created a new project with the angular cli and added `@rx-angular/template` via `ng add @rx-angular/template`.

This is the outcome of node_modules:

![image](https://user-images.githubusercontent.com/4904455/108897334-455ccd80-7616-11eb-86e4-1412da997a23.png)

The file which is missing for you is actually here. A build with ivy is working as well. Can you provide some more information please? I will check out your project now

**Comment by hoebbelsB** [comment]
@lincolnthree I have checked out your project and it builds fine? btw. next time please share a git repository... I actually wonder why I just trusted your zip file :O

**Comment by hoebbelsB** [comment]
sorry, I obviously didn't catch that this is an issue with the language service itself. I've tested it with webstorm. @Karnaukhov-kh has some more insights about this!

**Comment by Karnaukhov-kh** [comment]
Hi @lincolnthree. I'm not having any issues with build itself (in new project or your example).

But having some issues during ngcc  when "Angular: Experimental Ivy" enabled in language service settings. The thing is that you need to enable it to get proper hints in your templates. In ViewEngine based service it will show all custom structural directives and pipes as red (not recognized).

We will investigate this. It might be either issue with ng-packagr on our side, ng-packagr itself or experimental language service in VScode. 

We'll be back as soon as we get more info.

**Comment by lincolnthree** [comment]
@Karnaukhov-kh Thanks! I reported this to the vscode-angular folks first, but they seemed to think the issue was on this end, so I figured I'd tie the loose ends together :) ngcc really shouldn't crash on this IMO, but... here we are. Hope this helps, and thanks for the great library & support.

**Comment by hoebbelsB** [comment]
resolved with #533
